host/scripts/dss_host.py

```diff
@@ -445,13 +454,14 @@ def subnet_drive_map():
         for dev in subsys["Paths"]:
             name = '/dev/' + dev["Name"] + 'n1'
             transport = dev["Transport"]
-            addr = re.search(r"traddr=(\S+)", dev["Address"]).group(1)
-            subnet = getSubnet(addr)
+            if transport == "rdma":
```

Comment:
What about non-rdma path? Do we need to handle?

Reply:
I asked @hroogi for his feedback about that this morning. I'll let him elaborate if necessary, however currently TCP subsystem is not working, so in his opinion it's ok to explicitly only handle RDMA subsystems (for now).